ammo-fe/src/util/NetUtil/__tests__/test.ts

```diff
@@ -0,0 +1,16 @@
+import urlWithoutOrigin from 'util/NetUtil';
+
+describe('NetUtil', () => {
+    describe('urlWithoutOrigin', () => {
```

Comment:
Why two describes ?

Reply:
First describe is the class / file, second describe*s* are the tested methods of said file